### DISCUSSION TITLE
Implement SCRIPTS Wordpress

### DISCUSSION THREAD

**Discussion by manuelgamelas** [tomickigrzegorz/leaflet-examples#78] [General]
Hello, 

how can i implement this scripts in Wordpress+Elementor, can u help?
Best regard's

**Comment by tomickigrzegorz** [comment]
Hi,
Unfortunately, I don't have time and I don't know anything about Elementor.
Maybe look for a leaflet plugin that you can use with WordPress. This will be easier to implement than writing something of your own.

Good luck

**Comment by gergo18** [comment]
You can add your scripts with any helpre plugin. Code snippets or something else wp code maybe
